stream-api/src/main/kotlin/fund/cyber/markets/api/common/IncomingMessagesHandler.kt

```diff
@@ -48,4 +64,6 @@ class IncomingMessagesHandler(
         super.onClose(webSocketChannel, channel)
     }
 
-}
\ No newline at end of file
+}
+
+fun Int.Companion.rand(from: Int, to: Int) = (Math.random() * (to - from) + from).toInt()
```

Comment:
Move to separate file, something like `Extensions.kt`

Reply:
+